js/api/PopulateLocalParseServer.js

```diff
@@ -0,0 +1,140 @@
+
+import { loadSurveyList, storeSurveys } from '../api/Surveys'
+import { loadForms, storeForms } from '../api/Forms'
+import { loadQuestions, storeQuestions } from '../api/Questions'
+import { loadTriggers, storeTriggers } from '../api/Triggers'
+
+// Attempts to create interactive dummy data in a local server.
+export function initializeLocalParseData () {
+  if ( Store.server === 'remote-test' ) {
+    loadSurveyList({})
+  } else {
+    loadSurveyList({}, createInitialParseData)
+  }
+}
+
+function createInitialParseData (error, results) {
+  if (error) {
+    console.warn(error)
+  } else if (results.length === 0) {
+    console.log('Creating initial Parse server data...')
+    for (i = 0; i < DummyData.surveys.length; i++) {
+      createSurvey(DummyData.surveys[i])
+    }
+  }
+}
+
+function createSurvey(surveyData) {
+  const Survey = Parse.Object.extend('Survey')
+  let newSurvey = new Survey()
+
+  newSurvey.set('title', surveyData.title)
+  newSurvey.set('user', surveyData.user)
+  newSurvey.set('createdAt', surveyData.created)
+
+  newSurvey.save(null, {
+    success: function(response) {
+      console.log('New Survey created with objectId: ' + response.id)
+      createForms(response)
+      storeSurveys(response)
+    },
+    error: function(response, error) {
+      console.warn('Failed to create Survey, with error code: ' + error.message)
+    }
+  })
+}
+
+function createForms(parentSurvey) {
+  const Form = Parse.Object.extend('Form')
+  let newForm = new Form()
+
+  newForm.set('survey', parentSurvey)
+  
+  newForm.save(null, {
+    success: function(response) {
+      console.log('New Form created with objectId: ' + response.id)
+      createQuestions(response)
+      createTriggers(response)
+      storeForms(response)
+    },
+    error: function(response, error) {
+      console.warn('Failed to create Form, with error code: ' + error.message)
+    }
+  })
+}
+
+function createQuestions(parentForm) {
+  const Question = Parse.Object.extend('Question')
+  for (var i = 0; i < DummyData.questions.length; i++) {
+    let newQuestion = new Question()
+    
+    newQuestion.set('text', DummyData.questions[i].text)
+    newQuestion.set('questionType', DummyData.questions[i].questionType)
+    newQuestion.set('properties', DummyData.questions[i].properties)
+    newQuestion.set('form', parentForm)
+
+    newQuestion.save(null, {
+    success: function(response) {
+        console.log('New Question created with objectId: ' + response.id)
+        storeQuestions(response)
+      },
+      error: function(response, error) {
+        console.warn('Failed to create Question, with error code: ' + error.message)
+      }
+    })
+  }
+}
+
+function createTriggers(parentForm) {
+  const Trigger = Parse.Object.extend('Trigger')
+  let newTrigger = new Trigger()
+
+  newTrigger.set('triggerType', DummyData.trigger.triggerType)
+  newTrigger.set('properties', DummyData.trigger.properties)
+  newTrigger.set('form', parentForm)
```

Comment:
Not setup yet on server/admin but this relation should also be stored on the form.